osc-server/src/main/java/org/osc/core/broker/service/securitygroup/BindSecurityGroupService.java

```diff
@@ -218,49 +244,48 @@ private boolean isServiceSelected(List<VirtualSystemPolicyBindingDto> servicesTo
 		return false;
 	}
 
-	protected void validateAndLoad(EntityManager em, BindSecurityGroupRequest request) throws Exception {
+	protected void basicValidation(EntityManager em, BindSecurityGroupRequest request) throws Exception {
 		BindSecurityGroupRequestValidator.checkForNullFields(request);
```

Comment:
checknullfields can use the request to figure out which fields like order based on sfc or binding